code/modules/pai/pai.dm

```diff
@@ -159,12 +159,7 @@
 	QDEL_NULL(internal_gps)
 	QDEL_NULL(newscaster)
 	QDEL_NULL(signaler)
-	if(!QDELETED(card) && loc != card)
-		card.forceMove(drop_location())
-		// these are otherwise handled by paicard/handle_atom_del()
-		card.pai = null
-		card.emotion_icon = initial(card.emotion_icon)
-		card.update_appearance()
```

Comment:
@jlsnow301 Please advise, this should never have been on Destroy() (forceMove is an observable side effect) so I'm removing this, but not really sure what we're losing from this.

Reply:
pais outside of their card are set to drop the card, hopefully that behavior remains, but the rest of this should be okay to delete

Reply:
@Mothblocks behavior still good? destruction/death maybe?